pages/characters/[characterId].tsx

```diff
@@ -1,6 +1,9 @@
 import {
```

Comment:
I think Ben's idea was to completely remove individual character pages. What do you think of that @dan13ram ?

Reply:
Removed in 5fff41d32f2580a90e7ca648147de5f3bd57535c